HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServices.json

```diff
@@ -30,34 +66,660 @@
+                "expression": "\"HomeHealthServicesPrepopulation\".XXXXXXXXXXXXXXX_freq"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "9.3",
+          "text": "Duration",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".XXXXXXXXXXXXXXX_duration"
+              }
+            }
+          ]
+        }
+      ]
     },
     {
-      "linkId": "2.5",
-      "text": "Date of Face-To-Face Evaluation",
-      "type": "date",
-      "required": true,
+      "linkId": "10",
+      "text": "Other Services",
+      "type": "group",
+      "repeats": true,
       "extension": [
         {
-          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
-          "valueExpression": {
-            "language": "text/cql",
-            "expression": "\"HomeHealthServicesPrepopulation\".Today"
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/questionnaire-item-control",
+                "code": "gtable"
+              }
+            ]
           }
         }
+      ],
+      "item": [
+        {
+          "linkId": "10.1",
+          "text": "Service Name",
+          "type": "open-choice",
+          "required": true,
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.104",
+          "extension": [
+            {}
+          ]
+        },
+        {
+          "linkId": "10.2",
+          "text": "Frequency",
+          "type": "choice",
+          "required": false,
+          "answerValueSet": "http://hl7.org/fhir/ValueSet/timing-abbreviation",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".XXXXXXXXXXXXXXX_freq"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "10.3",
+          "text": "Duration",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".XXXXXXXXXXXXXXX_duration"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "11",
+      "text": "Signature, name, signature date, NPI, address, and telephone number of ordering physician",
```

Comment:
We could use Patient-info subquestionnaire with expand to add phone number

Reply:
good idea

Reply:
We can use provider-signature subquestionnaire then expand it with phone number. Is there any reason not using the subquestionnaire?